include/CMakeLists.txt

```diff
@@ -1,13 +1,60 @@
 #install commands for headers
-FILE(GLOB files ${CMAKE_CURRENT_SOURCE_DIR}/*.h*)
-INSTALL(FILES ${files} DESTINATION include)
-# Copy the file to directory matching the install directory
-file(COPY ${files} DESTINATION "${PROJECT_BINARY_DIR}/include")
 
-FILE(GLOB files ${CMAKE_CURRENT_SOURCE_DIR}/*.inl)
-INSTALL(FILES ${files} DESTINATION include)
-# Copy the file to directory matching the install directory
-file(COPY ${files} DESTINATION "${PROJECT_BINARY_DIR}/include")
+set(H_HEADERS
+  amp_math.h
+  amp_short_vectors.h
+  amp.h
+  grid_launch.h
+  grid_launch.hpp
+  hc_am.hpp
+  hc_defines.h
+  hc_math.hpp
+  hc_printf.hpp
+  hc_rt_debug.h
+  hc_short_vector.hpp
+  hc.hpp
+  hsa_atomic.h
+  kalmar_aligned_alloc.h
+  kalmar_buffer.h
+  kalmar_cpu_launch.h
+  kalmar_exception.h
+  kalmar_index.h
+  kalmar_launch.h
+  kalmar_math.h
+  kalmar_runtime.h
+  kalmar_serialize.h
+  pinned_vector.hpp)
+
+set(INL_HEADERS
+  hc_norm_unorm.inl
+  hc_short_vector.inl
+  kalmar_short_vectors.inl)
+
```

Comment:
is there a way to automatically generate these lists instead of constructing them manually?